.github/workflows/pr-deploy.yaml

```diff
@@ -318,51 +299,39 @@ jobs:
           kubectl create secret generic coder-db-url -n pr${{ env.PR_NUMBER }} \
             --from-literal=url="postgres://coder:coder@coder-db-postgresql.pr${{ env.PR_NUMBER }}.svc.cluster.local:5432/coder?sslmode=disable"
 
+      - name: Create a kubeconfig for the workspace
```

Comment:
Like I said above idk if this is really necessary, you can add `serviceAccountName` to the pod spec and the SA will be mounted. I'm pretty sure kubectl will also just see the SA immediately without any fuss, but not entirely sure. Might still have to set KUBECONFIG to point to the auto-mounted config (not one you make yourself)

Reply:
Can we chat about this. I am not exactly sure how I can go this path.

Reply:
Thanks this worked.